chrome/userContent.css

```diff
@@ -2,3 +2,4 @@
 @import url("content/common.css");
 @import url("content/newtab.css");
 @import url("tweaks/sidebery.css") (-moz-bool-pref: "uc.tweak.theme.sidebery");
+@import url("tweaks/sidebery-collapsed.css") (-moz-bool-pref: "uc.tweak.collapse-sidebery");
```

Comment:
If you move the collapsed Sidebery code to the `sidebery.css` file then the import should be removed.